src/entity/components/Interactable.ts

```diff
@@ -7,19 +7,25 @@ export type Interactable = ECSComponent<
   InteractableBrand,
   {
     text: string;
+    pickupRadius: number;
```

Comment:
Hm. I think it might be more interesting to model this via a "Name" component. We might want to display names for more things than just intractable, for example, logs (orc does thing).